ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/KsqlQueryEndpointMessage.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.confluent.ksql.rest.server.resources.Errors;
+
+import java.util.Collections;
+import java.util.List;
+
+public class KsqlQueryEndpointMessage extends KsqlStatementErrorMessage {
```

Comment:
Do we need this class? i.e., can we not just construct a`KsqlStatementErrorMessage` with the appropriate message? Doesn't seem to be adding much